File: langgraph/data/equiv/159.py
def consume(quantity, requirement, stock):
    if requirement <= stock:
        updated_quantity = quantity + requirement
        leftover_stock = stock - requirement
    else:
        updated_quantity = quantity + stock
        leftover_stock = 0

    return [updated_quantity, leftover_stock]
# method name: eat -> consume
# parameter name: number -> quantity, need -> requirement, remaining -> stock
# replaced if-else with a more descriptive block structure
# renamed variables for clarity: number -> updated_quantity, remaining -> leftover_stock

File: langgraph/result/qwen/32b-eq2js/159.js
function consume(quantity, requirement, stock) {
    let updatedQuantity;
    let leftoverStock;

    if (requirement <= stock) {
        updatedQuantity = quantity + requirement;
        leftoverStock = stock - requirement;
    } else {
        updatedQuantity = quantity + stock;
        leftoverStock = 0;
    }

    return [updatedQuantity, leftoverStock];
}